frontend/app/components/SessionIndividual/logic.js

```diff
@@ -0,0 +1,11 @@
+import withProps from 'recompose/mapProps';
+
+const withLogic = withProps(
+  ({ sessions, match: { params: { sessionName } } }) => {
+    return {
+      session: sessions.find(({ urlString }) => urlString === sessionName),
```

Comment:
Seems the only reason this is necessary is to support the 404 logic, which probably doesn't belong here anyway?